capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java

```diff
@@ -470,7 +470,7 @@ private void showFragment() {
     }
 
     private void showOnboardingIfRequested() {
-        if (shouldShowOnboarding()) {
+        if (shouldShowOnboarding() && !mAddPages) {
             startOnboardingActivity();
```

Comment:
[Clarification] what's the purpose of mAddPages?

Reply:
@zladzeyka mAddPages is a variable that tells us if Camera Screen is launched from MultiPageReview Screen